R/utils.R

```diff
@@ -4,12 +4,20 @@
 #' @returns Writes configuration lines to `file_path`.
 #'
 #' @examples
-#' \dontrun{
+#' if (interactive()) {
 #'   use_box_lsp()
 #' }
 #'
 #' @export
 use_box_lsp <- function(file_path = ".Rprofile") {
+  response <- readline(
+    glue::glue("Would you like to create or modify the `{file_path}` file? (yes/No) ")
```

Comment:
I feel being more specific with the action would be better for the UX. In case when .Rprofile exists, you could ask for modification and mention, that the lines will be added and the original part will be preserved. And in the case when there is no Rprofile, that it will be created.